run/logging-manual/main.py

```diff
@@ -0,0 +1,64 @@
+# Copyright 2019 Google, LLC.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from flask import Flask, request
+import json
+import os
+import sys
+
+app = Flask(__name__)
+
+
+@app.route('/', methods=['GET'])
+def index():
+    project = os.environ['GOOGLE_CLOUD_PROJECT']
+
+    # [START run_manual_logging]
+    # Uncomment and populate this variable in your code:
+    # project = 'The project ID of your Cloud Run service';
+
+    # Build structured log messages as an object.
+    global_log_fields = dict()
+
+    # Add log correlation to nest all log messages
+    # beneath request log in Log Viewer.
+    trace_header = request.headers.get('X-Cloud-Trace-Context')
+
+    if trace_header and project:
+        trace = trace_header.split('/')
+        global_log_fields['logging.googleapis.com/trace'] = (
+            f"projects/{project}/traces/{trace[0]}")
+
+    # Complete a structured log entry.
+    entry = dict(severity='NOTICE',
```

Comment:
this:

`entry = dict(severity='NOTICE', ... **global_log_fields)
`

seemed more elegant than: 

```
entry = {"severity" = "NOTICE", .... }
entry.update(global_log_fields)
```
In short, wanted to be able to combine the dictionaries in one move, rather than two.